blocks/header/header.css

```diff
@@ -76,8 +90,22 @@ header nav.nav-middle p {
   margin: 0;
 }
 
-header nav.nav-middle a {
-  display: flex;
+
+header nav.nav-middle h6 {
```

Comment:
this change will also impact the normal header, is that correct ?
as nav-middle is also present there

Reply:
nav-middle h6 is not present in normal header doc. So wont be affecting it.